dataactbroker/handlers/aws/session.py

```diff
@@ -198,18 +183,27 @@ def open_session(self, app, request):
         app -- (Flask) the Flask applcation
         request -- (Request)  the request object
 
-        implements the open_session method that pulls or creates a new DynamoSession object
+        implements the open_session method that pulls or creates a new UserSession object
 
         """
         sid = request.headers.get("x-session-id")
         if(sid and SessionTable.doesSessionExist(sid)):
             if SessionTable.getTimeout(sid)> toUnixTime(datetime.utcnow()):
-                return DynamoSession(initial=SessionTable.getData(sid),sid=sid)
+                session_dict =  UserSession()
+                # Replace single quotes with double and read as dict, also need to correct True, False, and None for json
+                data_string = SessionTable.getData(sid).replace("'",'"').replace("True","true")\
+                    .replace("False","false").replace("None","null")
```

Comment:
This is the beginning of a custom serialization/deserialization protocol. Why not just store the data in JSON? Also, I'm pretty sure you can kick all that work over to Sqlalchemy if using [JSON](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.JSON) or [PickleType](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.PickleType)